### ISSUE TITLE
Submission ID not added to name of submitted form.

### ISSUE THREAD

**Issue by mrtomtom** [verbb/formie#374]
When I set up a form to have the submission ID in the title, it does not appear on the form list. In the screenshot the title in the submission is with first name and last name. On the syubmission list and exported date they both only include the first and last name.

<img width="818" alt="Screen Shot 2021-03-03 at 11 19 53 pm" src="https://user-images.githubusercontent.com/339642/109804967-fd255700-7c76-11eb-832a-5a4510a21959.png">

A secondary suggestion is that in the layout options for the table an option to have the submission ID as one of the columns.

**Comment by mrtomtom** [comment]
Sorry this should be listed as a bug not a feature request

**Comment by mrtomtom** [comment]
Also, the submission ID is not displayed anywhere on the edit page. So if you provide the submission number in the front end to the user who submits, the admin has no way to look up the submission in the back end.

**Comment by engram-design** [comment]
All these items should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mrtomtom** [comment]
Great thanks. When is the next release scheduled?

**Comment by engram-design** [comment]
In 1-2 days I'd say.

**Comment by engram-design** [comment]
Should be fixed in [1.3.22](https://github.com/verbb/formie/releases/tag/1.3.22)
